### PR TITLE
Remove unused variables for using DJ-Registry

### PR DESCRIPTION
**Pull request by tai271828**
The commit 4c1a1fc400d00ad4be963c902d9a92325804d08e introduced the usage
of DJ-Registry so we do not need the corresponding variables in the
configuration file. This commit follows up the the introducing of
DJ-Registry and perform more comprehensive house cleaning.

# How to Test This Pull Request
1. Get ready to have the permission to access the Django administration of the development site e.g. your local site.
1. Create at least two users and setup one of them as a reviewer.
1. Follow the README of the review system. Change the variables for each review stage[1].
1. Check the `My PyCon` dashboard behavior is the same as what the README describes.

[1] A screenshot of my development Django administration console

Selection_166

### CODE REVIEW

**Comment by tai271828** [comment]
@Lee-W FYI, you may want to be aware of the update of README of review system. I only update the way to present the variables for each review stage. Please let me know if I made any mistake on the README or you observe anything wrong with code change.

**Comment by tai271828** [comment]
@yychen I am not sure if we really want to merge the code right now in the middle of proposal review process : ) Please feel free to defer the code landing.

**Comment by tai271828** [comment]
Okay I have to update the corresponding test cases as well for the CI failure.

```
==================================== ERRORS ====================================
___________ ERROR collecting proposals/tests/views/test_speakers.py ____________
../../../../proposals/tests/views/test_speakers.py:9: in <module>
    not settings.PROPOSALS_EDITABLE,
/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/django/conf/__init__.py:77: in __getattr__
    val = getattr(self._wrapped, name)
E   AttributeError: 'Settings' object has no attribute 'PROPOSALS_EDITABLE'
___________ ERROR collecting proposals/tests/views/test_speakers.py ____________
../../../../proposals/tests/views/test_speakers.py:9: in <module>
    not settings.PROPOSALS_EDITABLE,
/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/django/conf/__init__.py:77: in __getattr__
    val = getattr(self._wrapped, name)
E   AttributeError: 'Settings' object has no attribute 'PROPOSALS_EDITABLE'
```

**Comment by tai271828** [comment]
We may want to implement test jobs for the usage of DJ-Registry rather than just removing unused test jobs.

**Comment by yychen** [comment]
> 
> 
> We may want to implement test jobs for the usage of DJ-Registry rather than just removing unused test jobs.

I think I had already done it. Please take a look at https://github.com/pycontw/pycon.tw/pull/730/files and see if it is sufficient.

File: src/reviews/README.md [hunk 1]
```diff
@@ -17,46 +17,56 @@ There are several time periods during the whole Call for Proposals and Proposal
...
 7. Announcement of Acceptance
 
-Each time period has a set of settings that need to be set in src/pycontw2016/settings, or
+Each time period has a set of settings that need to be set in the Django administration dashboard, or
 some steps you might need to go through, which will be documented here.
 
 ### 1. Call for Proposals
 
-`settings.py`:
+In the administration dashboard, go to `Home > Django Registry > Entries` and you should have the following variables
```
Comment (Lee-W) [inline src/reviews/README.md:25]: How about also providing URL like `https://localhost:8000/2020/admin/registry/entry/`

**Comment by yychen** [comment]
It looks great to me :D Thanks a lot for the effort. More coming in the future, including a easy to use interface to setup these variables among different stages.

File: src/reviews/README.md (hunk 1 above)
Comment (yychen) [reply src/reviews/README.md:25]: Good advice, but I think in the development environment, the year is not necessary? maybe something like `https://localhost:8000/admin/registry/entry/`? need to test it out.

Comment (yychen) [reply src/reviews/README.md:25]: yep, confirmed, in local development environment, the `2020` is not required. But I'm curious does it vary in different environment? Can you check for us @Lee-W :D

Comment (tai271828) [reply src/reviews/README.md:25]: I incline to keep the description yearly-neutral. Your suggestion could be an example. What do you think?

Comment (tai271828) [reply src/reviews/README.md]: Fixed up according to your suggestion. You may want to review it again.

**Comment by tai271828** [comment]
> > We may want to implement test jobs for the usage of DJ-Registry rather than just removing unused test jobs.
> 
> I think I had already done it. Please take a look at https://github.com/pycontw/pycon.tw/pull/730/files and see if it is sufficient.

@yychen this is awesome! Then I will let you decide when we will merge the code.

File: src/reviews/README.md [hunk 2]
```diff
@@ -17,46 +17,60 @@ There are several time periods during the whole Call for Proposals and Proposal
...
+Each time period has a set of settings that need to be set in the Django administration dashboard, or
 some steps you might need to go through, which will be documented here.
 
 ### 1. Call for Proposals
 
-`settings.py`:
+In the administration dashboard, go to `Home > Django Registry > Entries` with corresponding `SLUG`.
+`SLUG` is the corresponding string of the conference evet, e.g. `pycontw-2020` for 2020.
+For example, if you are running the version of year 2020,
+you may also access the console via `https://<host address>:8000/2020/admin/registry/entry/`
```
Comment (yychen) [inline src/reviews/README.md:28]: I think you don't need the `2020` in the development environment, can you check it with your own environment @tai271828

Comment (tai271828) [reply src/reviews/README.md:28]: Confirmed: no `2020` in the development environment.

Comment (tai271828) [reply src/reviews/README.md]: Updated the corresponding session. Thanks!

**Pull request merged**